debian/changelog

```diff
@@ -1,3 +1,13 @@
+ostree (2017.10-1endless1) unstable; urgency=medium
```

Comment:
You may want to use `-0endless1` here in case debian catches up and we want to incorporate their `-1` version. Kinda theoretical since jenkins is going to make up a much newer upstream version, though...